html/webappapis/dynamic-markup-insertion/opening-the-input-stream/mutation-observer.window.js

```diff
@@ -0,0 +1,22 @@
+async_test(t => {
+  const frame = document.body.appendChild(document.createElement("iframe"));
+  t.add_cleanup(() => { frame.remove(); });
+  const originalHTMLElement = frame.contentDocument.documentElement;
+  assert_equals(originalHTMLElement.localName, "html");
+  let sync = true;
+  const observer = new frame.contentWindow.MutationObserver(t.step_func_done(records => {
+    assert_false(sync);
```

Comment:
I wouldn't assert that MutationObservers are async here; that's pretty out of scope.